helm/ako/templates/statefulset.yaml

```diff
@@ -50,11 +50,20 @@ spec:
               secretKeyRef:
                 name: avi-secret
                 key: username
+          {{ if .Values.avicredentials.password  }}
```

Comment:
If these values are not required in env anymore then, we can skip these variables from statefulset and configmap.

Reply:
removed